packages/blockchain-ui/enzyme/tests/TransactionOutput.test.tsx

```diff
@@ -0,0 +1,28 @@
+// tslint:disable no-unused-expression
+import React from 'react';
+import renderer from 'react-test-renderer';
+import TransactionOutput from '../../src/components/TransactionOutput/TransactionOutput';
+import chai from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+chai.should();
+chai.use(sinonChai);
+
+describe('TransactionOutput component', () => {
+    let mySandBox: sinon.SinonSandbox;
```

Comment:
Probably don't need any of the sandbox stuff in this file (ignore this if this will change in a future PR though).